src/store/getters/state-getters.ts

```diff
@@ -163,12 +163,14 @@ export const getBusinessId = (state: StateIF): string => {
   return state.stateModel.tombstone.businessId
 }
 
-/** The business identifier (aka incorporation number). */
-export const getCurrentBusinessName = (state: StateIF): string => {
-  // return the legal name from the name request for BEN IA corrections
-  // or the legal name of the business for others
+/** The original business identifier (aka incorporation number). */
+export const getOriginalBusinessName = (state: StateIF): string => {
+  // try to return the legal name from an IA's name request object
+  // try to return the legal name from the corrected filing's business object
+  // otherwise return the current legal name
   return (
     getOriginalIA(state)?.nameRequest?.legalName ||
+    getCorrectedFiling(state)?.business?.legalName ||
     getBusinessInformation(state).legalName
```

Comment:
Line 173 is to handle the case of a BEN IA that didn't have a NR (ie, a numbered BEN not a named BEN).